js/debug-bar.dev.js

```diff
@@ -102,4 +102,27 @@ wpDebugBar.Panel = function() {
 
 $(document).ready( wpDebugBar.init );
 
+/**
+ * Add the 'warning' class to the admin bar button for PHP errors encountered after the admin bar
+ * was initialized.
+ */
+$(document ).ready( function() {
+	var hasPHPErrors, hasNotices, button;
+
+	hasPHPErrors = $( '#debug-menu-link-Debug_Bar_PHP span.debug-bar-issue-warnings' );
```

Comment:
These could be optimized by doing `$( '#debug-menu-link-Debug_Bar_PHP' ).find( 'span.debug-bar-issue-warnings' )` or `$( 'span.debug-bar-issue-warnings', '#debug-menu-link-Debug_Bar_PHP' )`